lib/webenginesettings.cpp

```diff
@@ -102,6 +109,14 @@ void SailfishOS::WebEngineSettings::initialize()
     // Make long press timeout equal to the one in Qt
     engineSettings->setPreference(QStringLiteral("ui.click_hold_context_menus.delay"), QVariant(PressAndHoldDelay));
 
+    // Disable wasm_baselinejit to avoid crashes on lower memory devices; see JB#56635
+    quint64 totalMemory = getTotalMemory();
+    if (totalMemory < (2.5 * 1024 * 1024 * 1024)) {
+        // Devices with roughly 2Gb and lower
+        qDebug() << "Lower memory: disabling wasm_baselinejit";
```

Comment:
sailfish-components-webview deserves a logging categories. This would be info kind of a thing. Created a new task about logging categories (JB#56816).

Reply:
If you're aiming to keep this, there should "#include <QDebug>"

Reply:
I've added some logging categories, as an extra commit on top.